tests/migration_test.go

```diff
@@ -725,8 +721,11 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				pvName := "test-iscsi-dv" + rand.String(48)
 				// Start a ISCSI POD and service
 				By("Starting an iSCSI POD")
-				iscsiIP := tests.CreateISCSITargetPOD(cd.ContainerDiskEmpty)
-				_, err = virtClient.CoreV1().PersistentVolumes().Create(tests.NewISCSIPV(pvName, "2Gi", iscsiIP, k8sv1.ReadWriteMany, k8sv1.PersistentVolumeFilesystem))
+				iscsiTargetPod := tests.CreateISCSITargetPOD(cd.ContainerDiskEmpty)
```

Comment:
Why you changed to `cd.ContainerDiskEmpty` from  `cd.ContainerDiskCirros`? wouldn't that effect the ENV of the container in the ISCSI pod?

Reply:
Its `ContainerDiskEmpty` on master https://github.com/kubevirt/kubevirt/blob/master/tests/migration_test.go#L728

Reply:
Right, but that's the only place, Isaw you changed it in some more places to be `cd.ContainerDiskEmpty` the take a look here for example : 
https://github.com/kubevirt/kubevirt/blob/master/tests/migration_test.go#L1364

Reply:
Sorry this is my bad I will fix it, thanks!